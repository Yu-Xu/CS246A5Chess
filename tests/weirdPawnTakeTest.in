game human human
move a2 a4
move a7 a6
move a4 a5
move b7 b5
move a5 b6
move d7 d5
move d2 d3
move d5 d4
move c2 c4
move d4 c3
