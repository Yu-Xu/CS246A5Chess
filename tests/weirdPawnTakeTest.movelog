8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4 P_ _ _ _
3 _ _ _ _ 
2  PPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 _ppppppp
6 p_ _ _ _
5 _ _ _ _ 
4 P_ _ _ _
3 _ _ _ _ 
2  PPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 _ppppppp
6 p_ _ _ _
5 P _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  PPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 _ pppppp
6 p_ _ _ _
5 Pp_ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  PPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 _ pppppp
6 pP _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  PPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 _ p pppp
6 pP _ _ _
5 _ _p_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  PPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 _ p pppp
6 pP _ _ _
5 _ _p_ _ 
4  _ _ _ _
3 _ _P_ _ 
2  PP_PPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 _ p pppp
6 pP _ _ _
5 _ _ _ _ 
4  _ p _ _
3 _ _P_ _ 
2  PP_PPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 _ p pppp
6 pP _ _ _
5 _ _ _ _ 
4  _Pp _ _
3 _ _P_ _ 
2  P _PPPP
1 RNBQKBNR
  abcdefgh
Black Wins
8 rnbqkbnr
7 _ p pppp
6 pP _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ pP_ _ 
2  P _PPPP
1 RNBQKBNR
  abcdefgh
