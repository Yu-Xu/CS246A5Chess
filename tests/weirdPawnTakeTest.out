8 rnbqkbnr
7 _ p pppp
6 pP _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ pP_ _ 
2  P _PPPP
1 RNBQKBNR
